src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -6,6 +6,28 @@ CHANGELOG
 
  * added support to autowire public typed properties in php 7.4
  * added support for defining method calls, a configurator, and property setters in `InlineServiceConfigurator`
+ * added possibility to define service with abstract argument in xml files
```

Comment:
added possibility to define abstract arguments for services
the rest should be removed as we don't add examples in changelog files.
please move the example to the description of the PR

Reply:
fixed